Gemfile.lock

```diff
@@ -444,10 +444,10 @@ GEM
     simplecov-html (0.12.3)
     simpleidn (0.2.1)
       unf (~> 0.1.4)
-    sinatra (3.0.4)
+    sinatra (3.0.5)
```

Comment:
A change here refers to zeitwerk which is sometimes concerning, it looks safe current usages though.